packages/Python/CONFIG

```diff
@@ -16,7 +16,7 @@ LIB_DEPENDS = zlib readline ncurses libressl libffi
 # and tries to run it on the host.
 patch-extra = touch $(srcdir)/Include/graminit.h $(srcdir)/Python/graminit.c
 
-CFLAGS += -fPIC
+CFLAGS +=-fPIC -std=c99
```

Comment:
Please leave the space after the +=

Reply:
Actually, it would be better style to use a second `CFLAGS +=` line for this, thus:
```make
CFLAGS += -fPIC
CFLAGS += -std=c99
```

Surprised to see that we need to explicitly say C99 in our flags!  This must be a change in 1.17.5?

Reply:
I was struggling trying to determine where the numpy build is picking up it's CFLAGS from; seems to have -fPIC twice for some reason. The main python build has std=c99 already but numpy doesn't (in either 1.17.5 or 1.16.4). The specific issue seems to be with the code for radix sort using loop initial declarations (radix sort was introduced in 1.17.0).